internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -59,11 +59,21 @@ type RuntimeConfig struct {
 type ECRImage struct {
 	RepoURL  string // RepoURL is the ECR repository URL the container image should be pushed to.
 	ImageTag string // Tag is the container image's unique tag.
+	Digest   string // The image digest.
 }
 
-// GetLocation returns the location of the ECR image.
+// GetLocation returns the ECR image URI.
+// If a tag is provided for the image than prioritize refering to the image via the tag.
+// If a digest is present then refer to the image via the digest.
+// Otherwise, use the "latest" tag.
```

Comment:
Maybe be more specific about it: currently the only use case is when doing `package` command with the default tag.